### PR TITLE
all: remove developer from (almost) everything

### PR DESCRIPTION
**Pull request by pedronis**

### CODE REVIEW

**Comment by pedronis** [comment]
picking up #761 to fix feedback and integration tests and other todos

File: snappy/binaries.go [hunk 1]
```diff
@@ -143,10 +138,7 @@ func addPackageBinaries(m *snapYaml, baseDir string) error {
 			continue
 		}
 
-		aaProfile, err := getSecurityProfile(m, app.Name, baseDir)
-		if err != nil {
```
Comment (zyga) [inline snappy/binaries.go]: Not sure if the context is okay but this cuts away the ``if err != nil { ... }`` code.

File: snappy/click_test.go [hunk 2]
```diff
@@ -397,15 +379,14 @@ export HOME="$SNAP_USER_DATA"
 
 # Snap name is: pastebinit
 # App name is: pastebinit
-# Developer name is: mvo
 
-ubuntu-core-launcher pastebinit.pastebinit pastebinit.mvo_pastebinit_1.4.0.0.1 /snaps/pastebinit.mvo/1.4.0.0.1/bin/pastebinit "$@"
+ubuntu-core-launcher pastebinit.pastebinit pastebinit_pastebinit_1.4.0.0.1 /snaps/pastebinit/1.4.0.0.1/bin/pastebinit "$@"
```
Comment (zyga) [inline snappy/click_test.go]: I strongly suspect this requires changes to the launcher but I'll let @jdstrand comment on this.

File: snappy/info_test.go [hunk 3]
```diff
@@ -80,8 +79,8 @@ type: os`)
 	}
 
 	// now remove the channel
-	storeMinimalRemoteManifest("app."+testDeveloper, "app", testDeveloper, "1.10", "Hello.", "")
-	storeMinimalRemoteManifest("fwk", "fwk", testDeveloper, "1.0", "Hello.", "")
+	storeMinimalRemoteManifest("app", testDeveloper, "1.10", "Hello.", "")
+	storeMinimalRemoteManifest("fwk", testDeveloper, "1.0", "Hello.", "")
```
Comment (zyga) [inline snappy/info_test.go]: This sounds like dead code. Why do we have a "framework"-specific test after killing developer/origin

**Comment by zyga** [comment]
Looks good to me. LGTM

File: snappy/click_test.go (hunk 2 above)
Comment (pedronis) [inline snappy/click_test.go]: @zyga don't we have integration tests that need the launcher to work?

**Comment by niemeyer** [comment]
Still LGTM.

**Comment by pedronis** [comment]
now integration tests pass locally with a new u-d-f

**Comment by pedronis** [comment]
retest this please

**Pull request merged**